Description: Fix failing test with python 3.11
 Changes to stdlib `random` necessitate updates to the class.
 .
 Thanks to Paul Wise for pointing out the patch.
Bug: #1023978
Applied-Upstream: https://github.com/scikit-learn/scikit-learn/commit/63f92d4adb61aed58d656544cc6caa9d68cb6065
--- a/sklearn/datasets/tests/test_lfw.py
+++ b/sklearn/datasets/tests/test_lfw.py
@@ -84,8 +84,8 @@
 
         for i in range(5):
             first_name, second_name = random_state.sample(FAKE_NAMES, 2)
-            first_index = random_state.choice(np.arange(counts[first_name]))
-            second_index = random_state.choice(np.arange(counts[second_name]))
+            first_index = np_rng.choice(np.arange(counts[first_name]))
+            second_index = np_rng.choice(np.arange(counts[second_name]))
             f.write(
                 (
                     "%s\t%d\t%s\t%d\n"
